packages/mountable/package.json

```diff
@@ -25,84 +25,73 @@
     "type": "git",
     "url": "https://github.com/whitphx/stlite.git"
   },
-  "devDependencies": {
-    "@babel/core": "7.12.3",
-    "@pmmmwh/react-refresh-webpack-plugin": "0.4.3",
+  "dependencies": {
```

Comment:
This has been changed from `devDependencies` to `dependencies`.
Check if it's OK.

Reply:
The current `devDependencies` was introduced at https://github.com/whitphx/stlite/commit/74d8561c33daa90597b03ec212ef83da9f395d15

Reply:
Fix it here too.